updating/updating-restricted-network-cluster/index.adoc

```diff
@@ -13,6 +13,11 @@ If the local container registry and the cluster are connected to the mirror regi
 
 A single container image registry is sufficient to host mirrored images for several clusters in the disconnected network.
 
+== Mirroring the OpenShift Container Platform image repository
+In order to perform an update in a disconnected environment, your cluster environment must have access to a mirror registry that contains the necessary images and resources for your targeted update. The following page contains instructions for mirroring images onto a repository in your disconnected cluster:
```

Comment:
Again, I'd specify "cluster" somewhere here. Also "in order to" is usually unnecessary. Recommend something like:

```suggestion
To update your cluster in a disconnected environment, your cluster environment must have access to a mirror registry that contains the necessary images and resources for your targeted update. The following page contains instructions for mirroring images onto a repository in your disconnected cluster:
```